src/index.js

```diff
@@ -7,9 +7,9 @@ const creator = opts => {
 
 	return {
 		postcssPlugin: 'postcss-place',
-		Declaration: (decl) => {
+		Declaration: (decl, { result }) => {
 			if (placeMatch.test(decl)) {
-				onCSSDeclaration(decl)
+				onCSSDeclaration(decl, { result: result })
```

Comment:
[nit]

```suggestion
				onCSSDeclaration(decl, { result })
```

Reply:
No string opinions here.
I would add it as it is more similar :)